src/proteins.jl

```diff
@@ -0,0 +1,21 @@
+# this file will probably change drastically soon
+
+pdbid = Node("3P08")
+pdbdescript = @lift [getpdbdescription($pdbid) |> keys |> collect, getpdbdescription($pdbid) |> values |> collect] |> combinedims
+prot = @lift begin BioStructures.downloadpdb($pdbid; pdb_dir = "$(pdbdir())"); BioStructures.readpdb($pdbid; pdb_dir = "$(pdbdir())") end
```

Comment:
Just a quick note that I changed the keyword arguments for `downloadpdb` in the latest release of BioStructures (v0.10.0) a few days ago to be a bit more concise. This line could become:
```
prot = @lift begin BioStructures.downloadpdb($pdbid; dir = "$(pdbdir())"); BioStructures.read("$(pdbdir())/$pdbid.pdb", PDB) end
```